tools/scylla-nodetool.cc

```diff
@@ -788,6 +788,117 @@ void move_operation(scylla_rest_client& client, const bpo::variables_map& vm) {
+    };
+
+    fmt::print(std::cout, "        {} {} files, {} total. Already {} {} files, {} total\n",
+            action_continuous,
+            total_count,
+            format_bytes(total_size, human_readable),
+            action_perfect,
+            done_count,
+            format_bytes(done_size, human_readable));
+
+    for (const auto& file_entry : files.GetArray()) {
+        const auto& file = file_entry["value"];
+        fmt::print(std::cout, "            {} {}/{} bytes({}%) {} {} idx:{}/{}\n",
+                rjson::to_string_view(file["file_name"]),
+                file["current_bytes"].GetInt(),
+                file["total_bytes"].GetInt(),
+                uint64_t(file["current_bytes"].GetDouble() / file["total_bytes"].GetDouble() * 100.0),
+                action_perfect,
+                target,
+                file["session_index"].GetInt(),
+                rjson::to_string_view(file["peer"]));
+    }
+}
+
+void netstats_operation(scylla_rest_client& client, const bpo::variables_map& vm) {
+    const auto mode = sstring(rjson::to_string_view(client.get("/storage_service/operation_mode")));
+    const bool human_readable = vm.count("human-readable");
+
+    fmt::print(std::cout, "Mode: {}\n", mode);
+
+    const auto streams_res = client.get("/stream_manager/");
+    if (streams_res.GetArray().Empty()) {
+        fmt::print(std::cout, "Not sending any streams.\n");
+    } else {
+        for (const auto& stream : streams_res.GetArray()) {
+            fmt::print(std::cout, "{} {}\n", rjson::to_string_view(stream["description"]), rjson::to_string_view(stream["plan_id"]));
+
+            for (const auto& session : stream["sessions"].GetArray()) {
+                const auto ip = rjson::to_string_view(session["peer"]);
+                const auto private_ip = rjson::to_string_view(session["connecting"]);
+                fmt::print(std::cout, "    /{}", ip);
+                if (ip != private_ip) {
+                    fmt::print(std::cout, " (using /{})", private_ip);
+                }
+                fmt::print(std::cout, "\n");
+                print_stream_session(session["receiving_summaries"], session["receiving_files"], "Receiving", "received", "from", human_readable);
+                print_stream_session(session["sending_summaries"], session["sending_files"], "Sending", "sent", "to", human_readable);
+            }
+        }
+    }
+
+    if (client.get("/storage_service/is_starting").GetBool()) {
+        return;
+    }
+
+    fmt::print(std::cout, "Read Repair Statistics:\n");
+    fmt::print(std::cout, "Attempted: {}\n", client.get("/storage_proxy/read_repair_attempted").GetInt());
+    fmt::print(std::cout, "Mismatch (Blocking): {}\n", client.get("/storage_proxy/read_repair_repaired_blocking").GetInt());
+    fmt::print(std::cout, "Mismatch (Background): {}\n", client.get("/storage_proxy/read_repair_repaired_background").GetInt());
+
+    const char line_fmt[] = "{:<25}{:>10}{:>10}{:>15}{:>10}\n";
```

Comment:
could put
```c++
constexpr auto line_fmt = "{:<25}{:>10}{:>10}{:>15}{:>10}\n";
```
so we can use 
```c++
fmt::print(std::cout, line_fmt, "...");
```
later on. with which, the code is simpler, and we can have compile-time format check.

Reply:
done in v2